src/NuGetRepack.Tests/VersionUpdaterTests.cs

```diff
@@ -54,29 +54,34 @@ public void TestPackages()
             var dir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
             Directory.CreateDirectory(dir);
 
-            string a_daily, b_daily, c_daily;
+            string a_daily, b_daily, c_daily, d_daily;
             File.WriteAllBytes(a_daily = Path.Combine(dir, TestResources.DailyBuildPackages.NameA), TestResources.DailyBuildPackages.A);
             File.WriteAllBytes(b_daily = Path.Combine(dir, TestResources.DailyBuildPackages.NameB), TestResources.DailyBuildPackages.B);
             File.WriteAllBytes(c_daily = Path.Combine(dir, TestResources.DailyBuildPackages.NameC), TestResources.DailyBuildPackages.C);
+            File.WriteAllBytes(d_daily = Path.Combine(dir, TestResources.DailyBuildPackages.NameD), TestResources.DailyBuildPackages.D);
 
             var a_pre = Path.Combine(dir, TestResources.PreReleasePackages.NameA);
             var b_pre = Path.Combine(dir, TestResources.PreReleasePackages.NameB);
             var c_pre = Path.Combine(dir, TestResources.PreReleasePackages.NameC);
+            var d_pre = Path.Combine(dir, TestResources.PreReleasePackages.NameD);
```

Comment:
@tmat The test being added is failing, even though it seems the content of the generated nuspec/nupkg in the temp folder are identical to the one in the nuspec/nupkg added to test project Resources. Byte stream still shows difference, but I am not sure how to investigate it further.

Reply:
I'll take a look